packages/api/src/webhooks/cannon.ts

```diff
@@ -222,6 +239,69 @@ export default class WebhookCannon {
     );
   }
 
+  async notifyDisabledWebhook(
+    trigger: messages.WebhookTrigger,
+    params?: any,
+    err?: any
+  ) {
+    try {
+      if (trigger.user.emailValid) {
```

Comment:
**nit:** You can invert this if condition since you return from the `else`, then you can reduce 1 level of indentation on the whole thing (just like on the if below)